py3status/modules/spaceapi.py

```diff
@@ -23,12 +25,15 @@
 import datetime
 import json
 import urllib.request
+import shlex
+import subprocess
 
 
 class Py3status:
     """
     """
     # available configuration parameters
+    button_action = 3
```

Comment:
I think `url_button` might work. I was mirroring the `github` module here. Changed.